### ISSUE TITLE
Page break setting in NVDA should be changed

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#11624]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:

Some programs - [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
template=feature_request.md)

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Some proThese returns are announced by NVDA because the punctuation setting in the punctuation/symbols dialog for page break is none.  

A lot of users are probably very annoyed by this when they read exported files or receive files with hard returns as I imagine they do from Bookshare.  It is important that the default setting be changed to most or all.

**Comment by leonardder** [comment]
Could you please edit your message and fill in the bug template? What steps do you perform to reproduce the issue? What do you expect, and what's happening instead?

**Comment by Gene703122** [comment]
I'm new to Github and I sent an incorrect report.  I tried to present 
information but it wasn't there. You should see another properly filled out, 
I hope, report I created a little later.  I didn't see a way to remove the 
first one.

Gene
-----Original Message-----

**Comment by bhavyashah** [comment]
Based on https://github.com/nvaccess/nvda/issues/11624#issuecomment-694782369, I suggest the closure of this ticket.

**Comment by Adriani90** [comment]
We didn't get clear steps to reproduce for this and the issue template is not filled correctly. I am closing as abandoned since I cannot reproduce and we didn't get an update from the initial author.
